### ISSUE TITLE
Question: extractCss & lazyModules

### ISSUE THREAD

**Issue by mburger81** [kyubisation/angular-server-side-configuration#50]
When we run angular-server-side-configuration in version 3.x.x with ng11 we see this log
```
ngssc: Detected variant 'NG_ENV' with variables 'MAP_ZOOM_LEVEL, SOCKET_SERVER, MAP_DEFAULT_X, MAP_DEFAULT_Y, CALL_SERVER, API_SERVER, TIMEZONE'
Option "extractCss" is deprecated: Deprecated since version 11.0. No longer required to disable CSS extraction for HMR.
Option "lazyModules" is deprecated: 'SystemJsNgModuleLoader' is deprecated, and this is part of its usage. Use 'import()' syntax instead.
```
Why we can see this deprecated option if we don't use them? At the first look we thought we still use the options in angular which isn't so.

So how we have to understand this message?

Thx for the clarification

**Comment by kyubisation** [comment]
```
Option "extractCss" is deprecated: Deprecated since version 11.0. No longer required to disable CSS extraction for HMR.
Option "lazyModules" is deprecated: 'SystemJsNgModuleLoader' is deprecated, and this is part of its usage. Use 'import()' syntax instead.
```
This is not part of angular-server-side-configuration. I would assume it is a message from your "build" builder.

**Comment by kyubisation** [comment]
I've been able to reproduce it on another project. I will investigate this more thoroughly.

**Comment by kyubisation** [comment]
This should be solved with 11.0.2
